src/components/Dashboard/TodoList.vue

```diff
@@ -1164,9 +1165,9 @@ export default class TodoList extends Mixins(
 
       case FilingTypes.CORRECTION:
         if (item.correctedFilingType === FilingNames.INCORPORATION_APPLICATION) {
-          // redirect to Edit web app to correct this Incorporation Application
-          const correctionUrl = `${this.editUrl}${this.getIdentifier}/correction?correction-id=${item.filingId}`
-          window.location.assign(correctionUrl) // assume URL is always reachable
+          // navigate to Edit web app to correct this Incorporation Application
+          const correctionUrl = `${this.editUrl}${this.getIdentifier}/correction/?correction-id=${item.filingId}`
```

Comment:
Having this / at the end of the route and before the params is "more correct" but both work.

Reply:
oh...interesting.  I never thought about it deeply before but I've always used it without the `/` at the end.

Reply:
The way I see it, `.../correction/` is a route, so make it look like a route.

Conversely, `.../correction` could be a route with a parameter at the end. (Yes, a route parameter, but still a bit different IMHO).

Another way to look at it is that we specify various URLs in our config/env and most all end with a /, so we're following convention here.
```
  "AUTH_WEB_URL": "https://dev.bcregistry.ca/business/auth/",
  "DASHBOARD_URL": "https://dev.bcregistry.ca/business/",
```

Reply:
makes sense